changelog/add-5141-dispute-evidence-charge-id-2

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Show charge id on payments details page.
```

Comment:
Might be worth adding info about why here – i.e. provide merchant with charge id in case it's needed for responding to a `duplicate` dispute.

Reply:
Good idea. Added the why 6563c7a30d98e3b9bf0a5e7962aa2599887bd7cd.